src/authInfo.ts

```diff
@@ -365,6 +369,17 @@ export class AuthInfo extends AsyncCreatable<AuthInfo.Options> {
     this.options = options;
   }
 
+  /**
+   * Returns the default instance url
+   *
+   * @returns {string}
+   */
+  public static async getDefaultInstanceUrl(): Promise<string> {
+    const config = await ConfigAggregator.create({});
```

Comment:
Why don't you use the new sync method? `ConfigAggregator.getValue('instanceUrl').value`